scripts/release.kts

```diff
@@ -49,29 +49,42 @@ File(path)
         .forEach { folder ->
             val path = folder.path
             val module = folder.name
+            val pomFile = "$path/pom-default.xml"
 
             val file: String
 
             val androidFile = folder
                     .listFiles { it -> it.name.contains("release") && it.extension == "aar" }
                     .firstOrNull()
 
-            if (androidFile != null) {
-                file = androidFile.path
+            val bomFile = folder
```

Comment:
@iker-barriocanal these changes will need to be updated in the craft release too, right?

Reply:
Yes, will update.